examples/remote-state/fixtures.tfvars

```diff
@@ -2,4 +2,4 @@ enabled = true
 
 stack_config_local_path = "./stacks"
 
-stack = "my-stack"
+stack = "my-stack.yaml"
```

Comment:
Hrmm... Should it contain the `.yaml`? I think no. I want to be opinionated that it's always ending in `.yaml`

In most other places we don't include it (for example imports)

Reply:
Or we can have another parameter like `stack_config` which is `%s.yaml`